src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="temp2"
+        title="OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=DELTA"
+      >
+        New Relic supports metrics in delta temporality instead of the default of cumulative.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="bucket-hist"
+        title="OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION=EXPONENTIAL_BUCKET_HISTOGRAM"
+      >
+        (Recommended) Histogram aggregation: Use exponential histogram instead of default explicit bucket histogram for better data compression.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="logs-otlp"
+        title="OTEL_LOGS_EXPORTER=otlp"
+      >
+        Enable log export over OTLP. Log export is disabled by default.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="gzip"
+        title="OTEL_EXPORTER_OTLP_COMPRESSION=gzip"
+      >
+        (Recommended) Setting gzip compression allows you to maximize the amount of data you can send per request. The New Relic maximum allowed payload size is 1MB (10^6 bytes). Gzip compression has good performance and lowers data egress.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="retry"
+        title="OTEL_EXPERIMENTAL_EXPORTER_OTLP_RETRY_ENABLED=true"
+      >
+        (Recommended) Enable experimental export retry to help cope with the unreliability of the internet.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="disabled-keys"
+        title="OTEL_EXPERIMENTAL_RESOURCE_DISABLED_KEYS=process.command_line"
+      >
+        (Recommended) This disables the `process.command_line` resource attribute which often exceeds the New Relic maximum attribute length limit and may contain sensitive information.
+      </Collapser>
+    </CollapserGroup>
+
+
+5. Continue in `newrelic-opentelemetry-examples/getting-started-guides/java/Uninstrumented` to launch the agent with the app:
+
+    <Callout variant="important">
+      Replace `path/to` with the location of the JAR file you downloaded earlier.
+    </Callout>
+    ```
+    java -javaagent:path/to/opentelemetry-javaagent.jar \
+          -jar ./build/libs/fibonacci-java.jar
+    ```
+6. Generate traffic from the application by opening a new terminal in the same directory and running the load generator:
+    ```
+    ./load-generator.sh
+    ```
+
+    <Callout variant="tip">
+      Alternatively, you can reach the endpoint in the browser at this URL: `http://localhost:8080/fibonacci?n=INSERT_A_VALUE`. Replace `INSERT_A_VALUE` with value from 1 to 90. To generate an error, insert an integer outside the valid range.
```

Comment:
```suggestion
      Alternatively, you can reach the endpoint in the browser at this URL: `http://localhost:8080/fibonacci?n=INSERT_A_VALUE`. Replace `INSERT_A_VALUE` with a value from 1 to 90. To generate an error, insert an integer outside the valid range.
```

Reply:
@zuluecho9 since I'm in the middle of making changes, I just did this in both instances.